### PR TITLE
Feat/add subtabs component

### LINKED ISSUE #15773: Unnecessary change of Sell/Buy/DCA buttons when collapsing sidebar
**Describe the bug**
Unnecessary change of Sell/Buy/DCA buttons when collapsing sidebar 

**Steps to reproduce:**
1. Navigate to Trading section
2. Collapse sidebar
3. Observe buy/sell buttons

**Info:**
 - Suite version: web 25.1.0 (e2860882d84f904cab9a204ec964d5725455fe89)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 2.0.33

**Screenshots:**

https://github.com/user-attachments/assets/d257909d-a7e2-460d-b779-2217c1f6924a

### PR DESCRIPTION
**Pull request by seibei-iguchi**
Screenshot 2024-12-10 at 15 20 59

### CODE REVIEW

File: packages/components/src/components/Subtabs/Subtabs.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ${withFrameProps}
+`;
+
+export type SubtabsProps = AllowedFrameProps & {
+    children: ReactNode;
+    size?: SubtabsSize;
+    activeItemId?: string;
+};
+
+const Subtabs = ({ activeItemId, size = 'medium', children, ...rest }: SubtabsProps) => {
```
Comment (jvaclavik) [inline packages/components/src/components/Subtabs/Subtabs.tsx]: I would slightly prefer name `SubTabs`  🤔

File: packages/components/src/components/Subtabs/Subtabs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </Column>
+    );
+};
+
+export const Subtabs: StoryObj = {
+    render: props => {
+        return <SubtabsApp {...props} />;
+    },
+    args: {
+        ...getFramePropsStory(allowedSubtabsFrameProps).args,
```
Comment (jvaclavik) [inline packages/components/src/components/Subtabs/Subtabs.stories.tsx]: I think sort of the objects matters here, so frame props shouldn't be the first one, I think it should be the opposite.

```
    args: {
        size: 'medium',
        ...getFramePropsStory(allowedSubtabsFrameProps).args,
    },

**Review by jvaclavik — APPROVED** [review]

File: packages/components/src/components/Subtabs/Subtabs.stories.tsx (hunk 2 above)
Comment (seibei-iguchi) [reply packages/components/src/components/Subtabs/Subtabs.stories.tsx]: Though it doesn't matter in this case, right?

Comment (jvaclavik) [reply packages/components/src/components/Subtabs/Subtabs.stories.tsx]: Why do you think? There is also `size: 'medium'`

Comment (seibei-iguchi) [reply packages/components/src/components/Subtabs/Subtabs.stories.tsx]: Oh, I get it now, sorry

**Pull request merged**